docs/GettingStarted.md

```diff
@@ -2024,16 +2018,30 @@ You can also set this priority manually to either drop a non-interesting trace o
 
 When not using [distributed tracing](#distributed-tracing), you may change the priority at any time, as long as the trace incomplete. But it has to be done before any context propagation (fork, RPC calls) to be useful in a distributed context. Changing the priority after the context has been propagated causes different parts of a distributed trace to use different priorities. Some parts might be kept, some parts might be rejected, and this can cause the trace to be partially stored and remain incomplete.
 
-If you change the priority, we recommend you do it as soon as possible - when the root span has just been created.
+If you change the priority, we recommend you do it as soon as possible: distributed tracing requests will include
+the sampling decision, thus setting the sampling decision late can cause downstream services to have a different sampling
+rate than the originating service trace.
```

Comment:
Hmmm, I think the newly added explanation ends up being a repeat that what was above, but I do like it better than the previous one. Lightly edit both of them together perhaps?

Reply:
I've reworded it, let me know if reads better now.

Reply:
👍 Left a tiny remaining suggestion.